ndc-client/src/models.rs

```diff
@@ -27,16 +27,23 @@ pub struct CapabilitiesResponse {
 }
 // ANCHOR_END: CapabilitiesResponse
 
+// ANCHOR: LeafCapability
+/// A unit value to indicate a particular leaf capability is supported.
+/// This is an empty struct to allow for future sub-capabilities.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, JsonSchema)]
+pub struct LeafCapability {}
```

Comment:
We should also update the `mdbook` to include this type in `types.md` please.

Reply:
Done